Problem#1_HashMap.txt

```diff
@@ -0,0 +1,93 @@
+// Time Complexity : 0(n)
+// Space Complexity : 0(n)
+// Did this code successfully run on Leetcode : Yes
+// Any problem you faced while coding this : No
+
+
+class MyHashMap
+{
+	ListNode[] nodes;
+    
+	/** Initialize your data structure here. */
+    public MyHashMap() {
+        nodes = new ListNode[10000];
```

Comment:
Not a smart way to allocate so much space. The hashing data structure is given to see if a student can handle collisions in an efficient manner